pkgs/development/python-modules/mypy/default.nix

```diff
@@ -12,6 +12,11 @@ buildPythonPackage rec {
     sha256 = "069i9qnfanp7dn8df1vspnqb0flvsszzn22v00vj08nzlnd061yd";
   };
 
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "typed_ast >= 1.4.0, < 1.5.0" "typed_ast >= 1.4.0, < 2"
```

Comment:
Maybe adding a comment that this should be removed on the future `mypy 0.920` release?

https://github.com/python/mypy/issues/11158

Reply:
I am working on the mypy update. I should be able to be done tomorrow.

Reply:
Nice @SuperSandro2000.

We will still need to apply this patch: https://github.com/python/mypy/pull/11531

However it should apply cleanly after update.

Reply:
We need to fetch unstable mypy because we need another patch to allows loading types in packages that use mypy and I don't have the time to rebase that patch on the latest release. Currently no hunk of the patch applies cleanly.

Reply:
https://github.com/NixOS/nixpkgs/pull/146055